src/main/resources/templates/fragments/header.html

```diff
@@ -19,9 +22,18 @@
         <a class="nav-link" href="todo">Todo list</a>
       </li>
     </ul>
+    <ul class="navbar-nav navbar-right">
```

Comment:
Added a log in and a sign up link to the nav bar because I saw there's a login page so I figured it will be good to have the log in and the sign up link on there